src/net.rs

```diff
@@ -641,6 +661,17 @@ pub fn dev_status() -> ProcResult<HashMap<String, DeviceStatus>> {
     Ok(map)
 }
 
+/// Returns basic network device statistics for all interfaces
+///
+/// This data is from the `/proc/{pid}/net/dev` file.
+///
+/// For an example, see the [interface_stats.rs](https://github.com/eminence/procfs/tree/master/examples)
+/// example in the source repo.
```

Comment:
This comment is not accurate, since the interface_stats example doesn't use `dev_status_with_pid`

Reply:
Sorry for late, I think you are right. There is no need to import a new package for just using the pid_t type. I will renew this pull request later.